noxfiles/dev_nox.py

```diff
@@ -0,0 +1,66 @@
+"""Contains the nox sessions for running development environments."""
+import nox
+from constants_nox import (
+    COMPOSE_FILE,
+    IMAGE_NAME,
+    INTEGRATION_COMPOSE_FILE,
+    RUN,
+    START_APP,
+)
+from docker_nox import build_local
+
+
+@nox.session()
+def reset_db(session: nox.Session) -> None:
+    """Reset the database."""
+    build_local(session)
+    session.notify("teardown")
+    session.run(*START_APP, external=True)
+    reset_db_command = (*RUN, "fidesctl", "db", "reset", "-y")
+    session.run(*reset_db_command, external=True)
+
+
+@nox.session()
+def api(session: nox.Session) -> None:
```

Comment:
When I stopped this the teardown didn't get called so the container was still running and I had to stop it manually. I stopped it with `Ctrl + c` so I'm not sure there is anything you can do about this.

Reply:
I tried this as well, I didn't force it (another `ctrl + c`) and the teardown worked. I confirmed that forcing it does break the nox session and prevents teardown from running